clubs/views.py

```diff
@@ -100,20 +102,21 @@ def perform_create(self, serializer):
             status=Club.PENDING
         )
         send_mail(
-            subject=email.CREATE_MAIL_SUBJECT,
-            message=email.CREATE_MAIL_BODY % {
+            subject=email.create_club.plaintext.subject.render(Context()),
+            message=email.create_club.plaintext.body.render(Context({
```

Comment:
yeah, good to have that documented, especially the "this is a 1.7 thing we need to do". Maybe we should add a special label to indicate things we do specifically for our old version of django...